osh/cmd_parse.py

```diff
@@ -385,9 +386,12 @@ def _MakeSimpleCommand(
     more_env = []  # type: List[EnvPair]
     _AppendMoreEnv(preparsed_list, more_env)
 
+    if not len(redirects):
```

Comment:
style nitpick: I think this  reads better as `if len(redirects) == 0` , although it's the same thing